tests/system/python/packages/test_multiple_assets.py

```diff
@@ -358,4 +380,4 @@ def test_multiple_assets_with_reconfig(self, reset_fledge, start_north, read_dat
         if not skip_verify_north_interface:
             assert old_ping_result['dataSent'] < new_ping_result['dataSent']
             _verify_egress(read_data_from_pi_web_api, pi_host, pi_admin, pi_passwd, pi_db, wait_time, retries,
-                           total_benchmark_services)
+                           total_benchmark_services, PER_BENCHMARK_ASSET_COUNT)
```

Comment:
PER_BENCHMARK_ASSET_COUNT is num_assets_per_service?
and you don't need it to be ALL UPPER CASE as this is  now having  local / function level scope.

Reply:
> PER_BENCHMARK_ASSET_COUNT is num_assets_per_service? 

Yes PER_BENCHMARK_ASSET_COUNT is num_assets_per_service.